backend/src/akvo/lumen/lib/tier_impl.clj

```diff
@@ -3,7 +3,9 @@
    [ring.util.response :refer [response]]))
 
 (defn all [tenant-conn]
-  (response {"tiers" {"standard" {"numberOfExternalDatasets" 5
+  (response {"tiers" {"unlimited" {"numberOfExternalDatasets" "*"
```

Comment:
I was not sure what would be best but thought  * made more sense from the UI point of view. Rendering "nil" (not sure that's what you suggest) makes me think something is broken.